src/main/java/Duke.java

```diff
@@ -1,10 +1,198 @@
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+
 public class Duke {
+
+    static int idx = 0;
+    static ArrayList<Task> tasks = new ArrayList<>();
+
     public static void main(String[] args) {
```

Comment:
Good main() method, most of the key functions have been extracted out as methods 👍